tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/amqp/AmqpExpiredMessageTest.java

```diff
@@ -108,9 +109,16 @@ public void testExpiryThroughTTL() throws Exception {
       connection = addConnection(client.connect());
       session = connection.createSession();
 
-      AmqpReceiver receiverDLQ = session.createReceiver(getDeadLetterAddress());
+      AmqpReceiver receiverDLQ = session.createReceiver(getDeadLetterAddress(), "_AMQ_ORIG_ADDRESS='" + getQueueName() + "'");
```

Comment:
Instead of disturbing an existing test to validate new functionality you should really be writing new tests that cover the feature in depth with both positive and negative validation tests to ensure that future changes don't break and that the feature works as intended.

Reply:
You mean, copy & paste the test, and do the same? What for? it's the same test being done.

that will just add more time (a few seconds) to the testsuite for no reason.

Reply:
I mean write a real set of tests that validate that it applied to a given property and not others etc.  It really isn't that hard and given how long the broker tests take I'm pretty sure nobody will break out the torches and pitchforks.  Although given what the broker is doing is creating illegal message content maybe the tests should check for that not happening instead.